contracts/src/v0.8/ccip/libraries/Client.sol

```diff
@@ -51,9 +54,25 @@ library Client {
     bool allowOutOfOrderExecution;
   }
 
+  struct SolExtraArgsV1 {
+    uint32 computeUnits;
+    SolanaAccountMeta[] accounts;
+  }
+
+  struct SolanaAccountMeta {
```

Comment:
This feels pretty expensive as we use 33 bytes per account. We could consider the writability as a bool encoding into a digit?